### ISSUE TITLE
Two children with the same key `undefined` In a Bare React Native App 

### ISSUE THREAD

**Issue by alexpchin** [PedroBern/react-native-collapsible-tab-view#72]
I'm seeing the error:

```
Warning: Encountered two children with the same key, `undefined`. Keys should be unique so that components maintain their identity across updates. Non-unique keys may cause children to be duplicated and/or omitted — the behavior is unsupported and could change in a future version.
```

**Comment by alexpchin** [comment]
Perhaps resolved in #69 But need to test in Bare workflow.

**Comment by PedroBern** [comment]
Where are you seeing this error? Which example?
